### ISSUE TITLE
0.7(98) App crash if try to use without Bluetooth

### ISSUE THREAD

**Issue by markoaamunkajo** [ruuvi/com.ruuvi.station.ios#425]
Description:

User disables bluetooth opens Ruuvi Station app, closes popup about Bluetooth disabled, goes to change sensor settings, kills app
Opens Ruuvi Station app again - app crashes

https://user-images.githubusercontent.com/50437378/121781056-474e9a80-cbcd-11eb-84cc-12ac8d5f93fb.mov

**Comment by markoaamunkajo** [comment]
I'm not 100% what is the trigger, I assume changing setting. Sensors are network sensors.

**Comment by rinat-enikeev** [comment]
seems you've caught multithreading issue. fixed

**Comment by markoaamunkajo** [comment]
0.7(101) Managed to reproduce.
It's a bit of a hit and miss. Switch Bluetooth off, change sensor settings, kill app, restart app, open charts, zoom in, kill app, restart app change sensor settings, kill app, crash on restart. Sometimes doesn't.

I think this is not preventing us from pushing but moving back to investigate if it can be fixed.

**Comment by markoaamunkajo** [comment]
Oh wait hold on not sure if I uninstalled previous version.

**Comment by priyonto** [comment]
@markoaamunkajo I have been investing this issue for a while and I was not able to reproduce this issue on my device. As you have mentioned, this seems like a hit or miss issue although I have followed the steps you have mentioned here additionally I have just tried to randomly use the app without Bluetooth, and force close and then open again. Worked okay until now.  

Device: iPhone 11 Pro Max, iOS 15.2.1

If we can't reproduce it, I believe we can close it and if we find it again later, we can look into that.
